matchms/filtering/metadata_processing/require_valid_annotation.py

```diff
@@ -1,13 +1,63 @@
 import logging
 from matchms import Spectrum
 from matchms.filtering.filter_utils.smile_inchi_inchikey_conversions import (
-    convert_inchi_to_inchikey, convert_smiles_to_inchi, is_valid_inchi,
-    is_valid_inchikey, is_valid_smiles)
+    _check_fully_annotated, convert_inchi_to_inchikey, convert_inchi_to_smiles,
+    convert_smiles_to_inchi, is_valid_inchi, is_valid_inchikey,
+    is_valid_smiles)
+from matchms.filtering.metadata_processing.require_parent_mass_match_smiles import \
+    _check_smiles_and_parent_mass_match
 
 
 logger = logging.getLogger("matchms")
 
 
+def repair_not_matching_annotation(spectrum_in: Spectrum):
```

Comment:
This should be in a separate file `require_not_matching_anotation` like the other filters. According to the other files I was not looking in this python file for it.